<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>255</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>255</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QTCPClient</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="user_lbl">
        <property name="styleSheet">
         <string notr="true">color : black;
font-weight:bold;
font-size: 20px;
text-align:center;
</string>
        </property>
        <property name="text">
         <string>username</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="log_btn">
        <property name="styleSheet">
         <string notr="true">border-radius:10px;
background-color:#229ED9;
color:white;
font-size:18px;
font-weight:bold;
padding : 15px 0;</string>
        </property>
        <property name="text">
         <string>log in</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="pass_led">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">#input_signup_password
{
outline:none;
border:none;
border-bottom:1px solid lightgray;
background-color:white;
}
#input_signup_password:focus
{
border-bottom:1px solid #40ABE3;
}</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="user_led">
        <property name="styleSheet">
         <string notr="true">#input_signup_password
{
outline:none;
border:none;
border-bottom:1px solid lightgray;
background-color:white;
}
#input_signup_password:focus
{
border-bottom:1px solid #40ABE3;
}</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="pass_lbl">
        <property name="styleSheet">
         <string notr="true">color : black;
font-weight:bold;
font-size: 20px;
text-align:center;
</string>
        </property>
        <property name="text">
         <string>password</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="reg_btn">
        <property name="styleSheet">
         <string notr="true">border-radius:10px;
background-color:#229ED9;
color:white;
font-size:18px;
font-weight:bold;
padding : 15px 0;</string>
        </property>
        <property name="text">
         <string>register</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="time_lbl">
      <property name="styleSheet">
       <string notr="true">color:gray;
font-weight:400;
font-size:16px;
background-color:white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color:gray;
font-weight:400;
font-size:16px;
background-color:white;</string>
      </property>
      <property name="text">
       <string>The time you connected to the server :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
